### DISCUSSION TITLE
Import from / Export to spreadsheets

### DISCUSSION THREAD

**Discussion by clamyahoo** [nextcloud/bookmarks#2222] [Ideas]
Heya,

I would find it super if I could export bookmarks directly to a spreadsheet and vice-versa. Is that somehow feasable? The thing is, if I export bookmarks from NC and have only a html file, that doesn't help to work on the list. I would like to do this in an easy way because I have a long list of bookmarks, but in a completely different format.

I made a workaround with the help of ChatGPT: A Python app that allows you to convert automatically a bookmarks html file into an excel file. This file allows you to work on it. Afterwards, you can use another function in the app to convert it back into an html file that you can import into NC. The app is very basic and it would be great if there was a way to automize some more steps. Another thing is that you have to start it from the terminal (on Linux, I don't know if you can get it to work on other systems). I didn't make all the way through to convert it into an AppImage. It's only something to give you an idea. I hope it works for you.

[bookmark_converter.py.zip](https://github.com/user-attachments/files/16999833/bookmark_converter.py.zip)

Kind regards

**Comment by marcelklehr** [comment]
Hey @clamyahoo 
Thank you for sharing your suggestion and for your interest in improving the bookmarks app. We always appreciate hearing from users and getting new ideas.

That said, the feature you're proposing seems to address a very specific use case that may not align with the broader direction of the project. Given the scope we're focusing on and the need to keep the project maintainable for the long term, it's unlikely we'll be able to implement this at this time. It does sound like the python script you generated would be a decent workaround for the time being.

Thanks again for your input!
